app/volume.go

```diff
@@ -290,50 +288,54 @@ func (job *Job) backupAndCleanup() (err error) {
 
 	// Save current KubernetesStatus of Volume as a Label on the Backup. Run here because this should NOT be set on
 	// Snapshots.
-	v, err := job.GetVolume(job.volumeName)
+	v, err := job.GetVolume(volumeName)
 	if err != nil {
 		return err
 	}
 	// Cannot directly compare the structs since KubernetesStatus contains a slice which cannot be compared.
 	if !reflect.DeepEqual(v.Status.KubernetesStatus, types.KubernetesStatus{}) {
 		kubeStatus, err := json.Marshal(v.Status.KubernetesStatus)
 		if err != nil {
-			return errors.Wrapf(err, "BUG: could not convert volume %v's KubernetesStatus to json", job.volumeName)
+			return errors.Wrapf(err, "BUG: could not convert volume %v's KubernetesStatus to json", volumeName)
 		}
 		job.labels[types.KubernetesStatusLabel] = string(kubeStatus)
 	}
 
 	// CronJob template has covered the credential already, so we don't need to get the credential secret.
```

Comment:
Hmm, yeah, the comment there isn't actually needed because we're telling the `longhorn-manager` to handle the `Backups`, and that would have the credentials, not the `Cron Job`.